cmd/minishift/cmd/config/view.go

```diff
@@ -92,6 +95,6 @@ func configView() error {
 	// remove empy line at end
 	lines = lines[:len(lines)-1]
 	sort.Strings(lines)
-	fmt.Print(strings.Join(lines, "\n"))
+	fmt.Println(strings.Join(lines, "\n"))
```

Comment:
Join will not add `\n` end of the string so we need `Println` instead `Print`

- https://play.golang.org/p/E0Pnk7zvsG